ERCS/ERC-chain_id_rpc.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Chain-ID-Based RPC Provider URLs
+description: Standardizes how RPC providers expose node endpoints by chain-ID to simplify multichain integration.
+author: Soham Zemse (@zemse)
+discussions-to: <URL>
```

Comment:
Please create a discussions topic in Eth Magicians linking to this PR
https://ethereum-magicians.org/c/ercs/57